reportportal/v5/README.md

```diff
@@ -406,10 +406,15 @@ minio:
   installdep:
     enable: false
   endpoint: http://<minio-release-name>.default.svc.cluster.local:9000
+  region: null
   accesskey: <minio-accesskey>
   secretkey: <minio-secretkey>
 ```
 
+You can also use Amazon S3 instead of self-hosted MinIO through passing S3 endpoint and IAM user access key ID and secret to the RP_BINARYSTORE_MINIO_* env variables, which can be defined via the same parameters in values.yaml.  
```

Comment:
As i know, we don't support S3 transfer protocol. Minio still should be there as bridge to S3.
It would be better to re-phrase it as:
You can also use Amazon S3 `storage` instead of self-hosted MinIO's `storage` through passing S3 endpoint

Reply:
Re-phrased as requested